### DISCUSSION TITLE
Welcome to QField Discussions!

### DISCUSSION THREAD

**Discussion by m-kuhn** [opengisch/QField#2210] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you need a professional support please [contact us](mailto:info@opengis.ch).

**Comment by lp-dj** [comment]
Great! I have been waiting for such an "informal" way of communication.

Now I finally have a place where I can just ask straight out when the final QField 2.0 will be released, without hiding the question in another issue :D

I still have quite a few issues & feature requests in the pipeline, but thought it might make sense to wait for the supposedly imminent (?) version 2.0 and see what might already be done with it ;).

Oh, but you asked to introduce myself and what you want to do with the community:

Well, I work as an ecological surveyor and geoinformatician for whom GIS software is an everyday tool. I love QField for enabling me to empower other people to digitally capture valuable data in the field with their everyday devices and thus finally end the analog age of pen & paper in field ecology as well ;) Of course, I also use QField extensively in the field myself.

With this community I would like to prove once more that open source development is the best way for all involved. As long as I am not able to contribute my own pull request, I will at least try to contribute to the project via bug reports and hopefully useful enhancement suggestions ;)

Oh... and English is not my native language. So please excuse me if I write some nonsense somewhere ;)

**Comment by Oisin-JK** [comment]
Hi all
I'm an Arboricultural Consultant and use QField for tree surveys. I have dabbled in GIS for many years, using Manifold GIS, ArcPad, PostGIS. I write SQL, and a little code (VBA, vbscript, python).

My questions are: (1) is there any long term plan for QField to support scripting, say python, with form and field level events such as on open, on close, before update, after update etc; and (2) will Qfield forms ever support custom form design, or at least implenent the groups and columns feature avaialble in QGIS (even if this was limited to two columns). 

Regards O
